Back End/MemorizeWords/MemorizeWords/Infrastructure/Persistance/Repository/WordRepository.cs

```diff
@@ -129,17 +129,31 @@ private static void ValidationAddUpdateWord(WordAddRequest wordAddRequest)
 
         public async Task<List<QuestionWordResponse>> GetQuestionWordsAsync()
         {
-            var randomWords = await Queryable().Where(x => !x.IsLearned)
+            int randomNumber = new Random().Next(0, 21);
+
+            var randomWordsPLtoTR = await Queryable().Where(x => !x.IsLearned)
```

Comment:
await Queryable().Where(x => !x.IsLearned).OrderBy(x => Guid.NewGuid())

this line is used two times. take it to a single variable and use it for both queries.

Reply:
Yapıldı.